configparser.y

```diff
@@ -226,6 +238,8 @@ server_option:
     { cfg_parser->opt->hide_version = $2; }
   | VAR_HIDE_IDENTITY boolean
     { cfg_parser->opt->hide_identity = $2; }
+  | VAR_DROP_UPDATES boolean
+    { cfg_parser->opt->drop_updates = $2; }
```

Comment:
You cast (int) some bools above, but not this one.  And not hide_identity. I wonder what is going on.

Reply:
True. Removed them. Maybe we should eventually assign the result of `$2 != 0` instead so that the value is guaranteed to be either `1` or `0`, but it's ok for now.